org.eclipse.jdt.ui/ui refactoring/org/eclipse/jdt/internal/ui/refactoring/ExtractTempWizard.java

```diff
@@ -102,8 +104,11 @@ public void createControl(Composite parent) {
 			layouter.perform(label, text, 1);
 
 			addReplaceAllCheckbox(result, layouter);
-			addDeclareFinalCheckbox(result, layouter);
+			addReplaceAllInFileCheckbox(result, layouter);
 			addDeclareTypeVarCheckbox(result, layouter);
+			addDeclareFinalCheckbox(result, layouter);
```

Comment:
Don't change the previous order of existing fields.  It should be final then var option.